packages/@okta/i18n/src/properties/login_ru.properties

```diff
@@ -8,7 +8,7 @@ signin = \u0412\u043E\u0439\u0442\u0438 \u0432 \u0441\u0438\u0441\u0442\u0435\u0
 signup = \u0412\u0445\u043E\u0434
 remember = \u0417\u0430\u043F\u043E\u043C\u043D\u0438\u0442\u044C \u043C\u0435\u043D\u044F
 rememberDevice = \u0414\u043E\u0432\u0435\u0440\u044F\u0442\u044C \u044D\u0442\u043E\u043C\u0443 \u0443\u0441\u0442\u0440\u043E\u0439\u0441\u0442\u0432\u0443
-rememberDevice.timebased = \u041D\u0435 \u043F\u0440\u043E\u0432\u043E\u0434\u0438\u0442\u044C \u0430\u0443\u0442\u0435\u043D\u0442\u0438\u0444\u0438\u043A\u0430\u0446\u0438\u044E {0} \u043D\u0430 \u044D\u0442\u043E\u043C \u0443\u0441\u0442\u0440\u043E\u0439\u0441\u0442\u0432\u0435 \u0432 \u0442\u0435\u0447\u0435\u043D\u0438\u0435 \u0441\u043B\u0435\u0434\u0443\u044E\u0449\u0438\u0445 {0}
+rememberDevice.timebased = \u041D\u0435 \u043E\u0442\u043F\u0440\u0430\u0432\u043B\u044F\u0439\u0442\u0435 \u043C\u043D\u0435 \u0437\u0430\u043F\u0440\u043E\u0441/\u043E\u0442\u0432\u0435\u0442 \u043D\u0430 \u0430\u0443\u0442\u0435\u043D\u0442\u0438\u0444\u0438\u043A\u0430\u0446\u0438\u044E \u0432 \u0442\u0435\u0447\u0435\u043D\u0438\u0435 \u0441\u043B\u0435\u0434\u0443\u044E\u0449\u0438\u0445 {0} \u043C\u0438\u043D\u0443\u0442 \u043D\u0430 \u044D\u0442\u043E\u043C \u0443\u0441\u0442\u0440\u043E\u0439\u0441\u0442\u0432\u0435.
```

Comment:
@keithchin-okta https://oktainc.atlassian.net/browse/OKTA-414525
I tagged the bacon task with the JIRA but it didn't show up. Here is the Russian fix.